### DISCUSSION TITLE
webtoprint no image

### DISCUSSION THREAD

**Discussion by Jhorck15** [pimcore/pimcore#8297] [Q&A / Support]
Hi, good morning,

You do not have the styles or images, how can I solve this problem please

![image](https://user-images.githubusercontent.com/37857291/109392375-2437fd80-78f2-11eb-8330-e8770578dc78.png)

![image](https://user-images.githubusercontent.com/37857291/109392400-35810a00-78f2-11eb-9ebf-c017bf098a60.png)

**Comment by fashxp** [comment]
URLs to images and static resources need to be absolute (not relative).... that may be the reason.

**Reply by Jhorck15** [reply]
But styles don't work either

**Reply by fashxp** [reply]
styles  = static resources ... so all linked resources (images, css, js, fonts, ...) need to be absolute, not relative.

**Reply by Jhorck15** [reply]
![image](https://user-images.githubusercontent.com/37857291/109504474-f4afff00-7a71-11eb-9b9c-ae8fd3dd3122.png)

**Reply by macroplan** [reply]
Hello Jhorck15 
Did the provided code work for you?
